app/src/main/java/com/dreamisi/moviebase/FragmentMoviesList.kt

```diff
@@ -2,38 +2,36 @@ package com.dreamisi.moviebase
 
 import android.content.Context
 import android.os.Bundle
-import android.view.LayoutInflater
 import android.view.View
-import android.view.ViewGroup
 import android.widget.ImageView
 import androidx.fragment.app.Fragment
 
-class FragmentMoviesList : Fragment() {
+class FragmentMoviesList : Fragment(R.layout.fragment_movies_list) {
+
     private var fragmentMoviesListListener: FragmentMoviesListListener? = null
 
-    override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
-    ): View? {
-        val view = inflater.inflate(R.layout.fragment_movies_list, container, false)
-        view?.findViewById<ImageView>(R.id.movie_image)?.apply {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        view.findViewById<ImageView>(R.id.movie_image)?.apply {
             setOnClickListener {
                 fragmentMoviesListListener?.onFilmCardClicked()
             }
         }
-        return view
     }
 
-    override fun onAttach(context: Context) {
-        super.onAttach(context)
+
+    override fun onStart() {
+        super.onStart()
+        val context: Context? = context
         if (context is FragmentMoviesListListener) {
             fragmentMoviesListListener = context
         }
```

Comment:
Можно помоднее написать используя фичи котлина
```suggestion
        (context as? FragmentMoviesListListener)?.let { fragmentMoviesListListener = it }
```